src/p5.dance.js

```diff
@@ -1138,6 +1157,27 @@ module.exports = class DanceParty {
       title,
     };
 
+    let currentMeasure = Math.floor(this.getCurrentMeasure());
+    this.sprites_.forEach(sprite => {
+      if (sprite.alternatingMoveInfo) {
+        let alternatingMoveInfo = sprite.alternatingMoveInfo;
+        let val = Math.floor((currentMeasure - sprite.alternatingMoveInfo.start) / sprite.alternatingMoveInfo.cadence);
+        if (val % 2 == 0) {
+          if (sprite.alternatingMoveInfo.current != 1) {
+            this.changeMoveLR(sprite, sprite.alternatingMoveInfo.move1, -1);
+            sprite.alternatingMoveInfo = alternatingMoveInfo;
```

Comment:
Why do we need to reset this?

Reply:
`changeMoveLR` clears the `alternatingMoveInfo`, but we don't want to lose it in this case because we want to keep alternating